app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java

```diff
@@ -42,11 +38,10 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        ApplicationlessInjection
-                .getInstance(getActivity().getApplicationContext())
-                .getCommonsApplicationComponent()
-                .inject(this);
-
+        ApplicationlessInjection		
+ -                .getInstance(getActivity().getApplicationContext())		
+ -                .getCommonsApplicationComponent()		
+ -                .inject(this);
         // Load the preferences from an XML resource
```

Comment:
What are these "-" means? Can you make sure your code is compiled with no issue, before pushing it? Currently it doesn't.